packages/autometrics/src/platform.node.ts

```diff
@@ -112,6 +112,15 @@ export function getALSInstance() {
   }>();
 }
 
+/**
+ * Returns a boolean indicating whether a given path is the file-system root or not.
+ *
+ * @internal
+ */
+export function isRootPath(pathToCheck: str): string {
```

Comment:
oh i didn't actually pull down the code, but the type here should be `:string` instead of `:str`

Reply:
i've been writing a lot of python lately 😁